bass-midi/.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why is this directory only in bass-midi/?

Should this be here?
Should this be in guitar-hero-midi/?
Should this be gitignored and not pushed?